CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# Contributing to Bystro
+
+[WIP]
```

Comment:
I think we can remove this WIP marker for now, and just add changes as needed in future